pkg/report/table.go

```diff
@@ -134,7 +140,11 @@ func (tw TableWriter) setVulnerabilityRows(table *tablewriter.Table, vulns []typ
 		severityCount[v.Severity]++
 		lib := v.PkgName
 		if v.PkgPath != "" {
-			lib = fmt.Sprintf("%s (%s)", v.PkgName, v.PkgPath)
+			fileName := filepath.Base(v.PkgPath)
+			lib = fmt.Sprintf("%s (%s)", v.PkgName, fileName)
+			showMessageOnce.Do(func() {
+				log.Logger.Infof("Table result includes only package filenames. Use '--format json' subcommand to get full path to the package file.")
```

Comment:
Table result includes only package filenames. Use '--format json' option to get full path to the package file.

```suggestion
				log.Logger.Infof("Table result includes only package filenames. Use '--format json' option to get the full path to the package file.")
```